### PR TITLE
Update 05-reusable-components.md

### PR DESCRIPTION
**Pull request by cheesemacfly**
Updating the Autobinding section with the relevant part from the v0.13.0 update (https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding)

### CODE REVIEW

**Comment by facebook-github-bot-3** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-4** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: docs/docs/05-reusable-components.md [hunk 1]
```diff
@@ -254,10 +254,26 @@ Methods follow the same semantics as regular ES6 classes, meaning that they don'
...
+    this.tick = this.tick.bind(this);
+  }
+  tick() {
+    ...
+  }
+  ...
+}
+```
+
+Or using property initializers, there is a neat trick that you can use to accomplish this syntactically::
```
Comment (alexzherdev) [inline docs/docs/05-reusable-components.md]: Two colons at the end

**Comment by zpao** [comment]
Thanks, but we're not quite ready to recommend this pattern in the docs so I'm going to close out (blog post was the exception in order to explain at a point in time). Once that spec is slightly further along we'll update our documentation more broadly.

**Pull request closed without merging**